src/App.jsx

```diff
@@ -14,33 +17,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const IS_SORTED_ALPHABET = 'A';
+const IS_SORTED_LENGTH = 'L';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, { isSorted, isReversed }) {
+  const preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  switch (isSorted) {
+    case IS_SORTED_ALPHABET:
+      preparedGoods.sort((good1, good2) => good1.localeCompare(good2));
+      break;
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+    case IS_SORTED_LENGTH:
+      preparedGoods.sort((good1, good2) => good1.length - good2.length);
+      break;
+
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [isSorted, setIsSorted] = useState('');
```

Comment:
```suggestion
  const [sortType, setSortType] = useState('');
```
It's not boolean so prefix `is` is confusing